Documentation/install/upgrade.rst

```diff
@@ -181,7 +184,7 @@ Kubernetes resources are updated accordingly to version you are upgrading to:
    ``install/kubernetes/cilium/values.yaml`` and use it to regenerate the YAML
    for the latest version. Running any of the previous commands will overwrite
    the existing cluster's `ConfigMap` which might not be ideal if you want to
-   keep your existing `ConfigMap`.
+   keep your existing `ConfigMap` (see next option).
 
 Step 2: Option B: Preserve ConfigMap
 ------------------------------------
```

Comment:
Can you add `helm upgrade` in line 217 and remove the `config.keepCurrent=true` from line 225?